content/en/docs/faq/terminology.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "TLS Terminology"
+linkTitle: "TLS Terminology"
+weight: 50
+type: "docs"
+---
+
+With TLS being so widely deployed, terminology can sometimes get confused or be used to mean different things, and that reality
+combined with the complexity of TLS can lead to serious misunderstandings and confusion.
+
+### What does "publicly trusted" mean?
+
+Broadly speaking, a "publicly trusted" certificate is one that you can use on the public internet and expect
+that most reasonably up-to-date computers will be able to verify it using their system trust store.
+
+There isn't a single standard trust store containing certs which are "publicly trusted", but generally most
+of the commonly seen trust stores are similar. An example would be [Mozilla's CA Certificate Program](https://wiki.mozilla.org/CA).
```

Comment:
I looked to see how Python certifi library get the trusted CAs:

https://github.com/certifi/python-certifi/blob/8110050d8441d363c8057d5ad187c8ad5a862fc2/Makefile#L1-L2

They use https://mkcert.org/ which might be worth linking to?

Reply:
Which downloads trusted CAs from https://hg.mozilla.org/mozilla-central/raw-file/tip/security/nss/lib/ckfw/builtins/certdata.txt I think.

Reply:
My initial reaction here is that this might be out of scope for a terminology guide, because it's getting into pretty esoteric practices of customising trust stores. Is that fair?

These things would certainly be good to note in cert-manager/trust though.